tests/internal/test_module.py

```diff
@@ -339,3 +339,39 @@ def test_module_watchdog_dict_shallow_copy():
     assert original_modules == new_modules
 
     ModuleWatchdog.uninstall()
+
+
+@pytest.mark.skipif(sys.version_info < (3, 5), reason="requires python3.5 or higher")
+@pytest.mark.subprocess(out="ddtrace imported\naccessing lazy module\nlazy loaded\n")
+def test_module_watchdog_no_lazy_force_load():
```

Comment:
Should this test contain assertions for when `ddtrace imported`, `lazy loaded`, and `accessing lazy module` are logged? It's a little unclear how this test could fail